templates/common/_base/files/iptables-modules.yaml

```diff
@@ -0,0 +1,6 @@
+mode: 0644
+path: "/etc/modules-load.d/iptables.conf"
+contents:
+  inline: |
+    # Force-load legacy iptables so it is usable from pod network namespaces
+    ip_tables
```

Comment:
Is just this sufficient?  Aren't there a ton of other kernel modules related to netfilter?

Reply:
There are, but they get autoloaded as needed. The top-level module needs to be explicitly modprobed. (`/sbin/iptables` will try to do this itself if it sees that the module needs to be loaded, but that won't work when it's running from a container).

Reply:
@can you make sure nftables is modprobed first? I don't know about the ordering here, but we need to make sure nftables gets loaded before iptables for some corner cases, and if we're force-loading iptables, that may not happen in the right order. ICBW though.

Reply:
> we need to make sure nftables gets loaded before iptables for some corner cases

explain?

Reply:
@erig0 can you describe the issues around iptables/nftables? Is it module load order that matters, or is it just whatever process runs the first command that adds the hooks?

Reply:
I think the issue @dcbw is recalling is actually about NAT. In old kernels, if `iptables_nat` is loaded it would either a) cause nftables NAT to never work or b) cause nftables nat hook registration to fail - all depending on how old the kernel is. But [it was fixed back in 4.18](https://patchwork.ozlabs.org/project/netfilter-devel/cover/20180514214659.1757-1-fw@strlen.de/) (not 100% sure about kernel version).

The current behavior is:
  - both iptables_nat and nftables may register NAT hooks
  - the one that process first (i.e. highest priority hook) "wins" and the later NAT does nothing

Reply:
(Further clarifying from IRC: to the extent that any problem still exists, it happens if you create iptables-legacy NAT rules before creating iptables-nft NAT rules, _not_ if you just load `ip_tables` before `nf_tables`. So this PR is fine.)